pcweb/components/docpage/sidebar/sidebar.py

```diff
@@ -266,42 +266,49 @@ def get_prev_next(url):
     return None, None
 
 
-def sidebar_category(name, icon, color, index):
-    return rx.flex(
-        rx.button(
-            rx.icon(
-                tag=icon,
-                color="white",
-                fill="rgba(255, 255, 255, 0.25)",
-                size=20,
+def sidebar_category(name, url, icon, color, index):
+    return rx.link(
+        rx.flex(
+            rx.button(
+                rx.icon(
+                    tag=icon,
+                    color="white",
+                    fill="rgba(255, 255, 255, 0.25)",
```

Comment:
What's the difference between color and fill?

Reply:
For the icons color is the outline of the icon fill is the inside color, you can see slight differences if you check out the deployed sidebar item